astropy/modeling/fitting.py

```diff
@@ -2104,14 +2104,6 @@ def model_to_fit_params(model):
     return model_params, fitparam_indices, model_bounds
 
 
-@deprecated(
-    since="5.1",
-    message="private method: _model_to_fit_params has been made public now",
-)
-def _model_to_fit_params(model):
-    return model_to_fit_params(model)
-
-
```

Comment:
As @pllim pointed out, the `_filter_to_model_params` function should be removed at the same time as this. They are a matched pair of related functions.